applications/crb/minimal/poissonCRB.hpp

```diff
@@ -0,0 +1,179 @@
+#include <feel/feelcrb/modelcrbbase.hpp>
+#include <feel/options.hpp>
+#include <feel/feelmodels/modelproperties.hpp>
+
+namespace Feel
+{
+
+po::options_description
+makeOptions()
+{
+    po::options_description options( "Poisson" );
+    options.add_options()
+        ( "poisson.filename", Feel::po::value<std::string>()->default_value("poisson.json"),
+          "json file containing application parameters")
+        ( "poisson.gamma", po::value<double>()->default_value( 1e4 ), "penalisation term" )
+        ( "poisson.kappa", po::value<double>()->default_value(1), "conductivity" )
+        ( "poisson.flux", po::value<double>()->default_value(1), "flux intensity at the base" )
+        ( "poisson.N", po::value<int>()->default_value(100), "number of basis function to use" )
+        ;
+    return options;
+}
+
+AboutData
+makeAbout( std::string const& str = "poissoncrbmodel" )
+{
+    AboutData about( str.c_str() );
+    return about;
+}
+
+class Poisson : public ModelCrbBase<ParameterSpace<2>,
+                                    FunctionSpace<Mesh<Simplex<2> >,
+                                                  bases<Lagrange<1> > > >
+{
+public:
+    using super_type = ModelCrbBase<ParameterSpace<2>,
+                                    FunctionSpace<Mesh<Simplex<2> >,
+                                                  bases<Lagrange<1> > > >;
+    using value_type = double;
+    using element_type = super_type::element_type;
+    using parameter_type = super_type::parameter_type;
+    using mesh_type = super_type::mesh_type;
+    using mesh_ptrtype = super_type::mesh_ptrtype;
+    using prop_type = ModelProperties;
+    using prop_ptrtype = boost::shared_ptr<prop_type>;
+
+private:
+    mesh_ptrtype M_mesh;
+    prop_ptrtype M_modelProps;
+
+public:
+    int Qa();
+    int Nl();
+    int Ql( int l );
+
+    void initModel();
+    value_type
+    output( int output_index, parameter_type const& mu , element_type& u, bool need_to_solve=false);
+    boost::tuple<beta_vector_light_type, std::vector<beta_vector_light_type> >
+    computeBetaQ( parameter_type const& mu );
+
+}; // Poisson class
+
+int Poisson::Qa()
```

Comment:
what if the class is templated ? we can't split between cpp and hpp in this case.